BotBuilder-DotNet.ruleset

```diff
@@ -0,0 +1,601 @@
+﻿<?xml version="1.0" encoding="utf-8"?>
+<RuleSet Name="Rules for StyleCop.Analyzers" Description="Code analysis rules for StyleCop.Analyzers.csproj." ToolsVersion="15.0">
+  <Rules AnalyzerId="AsyncUsageAnalyzers" RuleNamespace="AsyncUsageAnalyzers">
+    <Rule Id="AvoidAsyncVoid" Action="Error" />
+    <Rule Id="UseConfigureAwait" Action="Error" />
+  </Rules>
+  <Rules AnalyzerId="Microsoft.CodeAnalysis.CSharp" RuleNamespace="Microsoft.CodeAnalysis.CSharp">
```

Comment:
All items below are `Code Analysis` issues treated as errors. Not `Source Analysis`, they are different in nature. This is not exactly what we want to achieve now.
At some point in the future, we want to treat everything as error, but right now only `Source Analysis`.

Reply:
Thx for your feedback, I will remove them.